src/common/definitions.js

```diff
@@ -533,10 +533,18 @@ const form4142 = {
             type: 'string'
           },
           treatmentDateRange: {
-            $ref: '#/definitions/dateRange'
+            type: 'array',
+            properties: {
```

Comment:
Is the idea here to make it an array of date ranges? If so, what do you think of calling this property `treatmentDateRanges` (plural to imply an array) and make it something like:
```js
treatmentDateRanges: {
  type: 'array',
  items: {
    $ref: '#/definitions/dateRange'
  }
}
```

Arrays require an `items` property and objects require a `properties` property. I don't think it'll work if we give an array `properties`.

Reply:
@cvalarida : Thanks for the feedback on the date range. Idea is to have date ranges to match the physical form. However we don't have UI to support date ranges in this phase. We already have the property configured in UI for transformation and in the backend for PDF generation. Based on your feedback, I made the change to use #/definitions/dateRange. Is this acceptable?